sources/api/apiserver/src/server/mod.rs

```diff
@@ -892,6 +964,12 @@ impl ResponseError for error::Error {
             UpdateLockOpen { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             ReportExec { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             ReportResult { .. } => StatusCode::INTERNAL_SERVER_ERROR,
+            DeserializeSettingsGenerator { .. } => StatusCode::BAD_REQUEST,
+            DeSerialize { .. } => StatusCode::BAD_REQUEST,
```

Comment:
Is there a more specific name we could give to this error? (What are we deserializing? Should it be an internal server error?)
```suggestion
            Deserialize { .. } => StatusCode::BAD_REQUEST,
```

Reply:
This has been renamed to `DeserialiseStrength`. This is used while parsing the pending strength that has been provided to the apiserver. Hence this can be kept as `BAD_REQUEST`